models/han/model.py

```diff
@@ -1,9 +1,8 @@
 import torch
 import torch.nn as nn
-import torch.nn.functional as F
 
-from han.sent_level_rnn import SentLevelRNN
-from han.word_level_rnn import WordLevelRNN
+from models.han import SentLevelRNN
+from models.han import WordLevelRNN
```

Comment:
from models.han.sent_level_rnn import SentLevelRNN?

Reply:
I fixed this in the latest commit. Thanks for pointing it out.